src/features/user/Account/components/AccountRecord.tsx

```diff
@@ -39,27 +28,37 @@ export function RecordHeader({
 }: HeaderProps): ReactElement {
   const { t, i18n } = useTranslation(['me', 'common']);
   const getRecordTitle = (): ReactElement => {
-    switch (record.type) {
-      case 'tree-donation':
-      case 'tree-gift':
-        if (record.unitType === 'tree')
-          return (
-            <p className={styles.top}>{`${getFormattedNumber(
-              i18n.language,
-              record.quantity
-            )} ${t(`me:${record.type}`, {
-              unitType: t(`common:${record.unitType}`, { count: 1 }),
-            })}`}</p>
-          );
-        else return <ProjectTitle record={record} />;
-      case 'funds-donation':
-      case 'bouquet-donation':
-      case 'conservation-donation':
-        return <ProjectTitle record={record} />;
+    let title: string;
+    switch (record.purpose) {
+      case 'trees':
+      case 'conservation':
+        // Sample title 1 => 5 Tree Gift . Yucatan,
+        // Sample title 2 => 2 m² Donation . Sumatra
+        title = `${getFormattedNumber(i18n.language, record.quantity)} ${t(
+          `common:${record.unitType}`,
+          { count: 1 }
+        )} ${
+          record.details.giftRecipient ? t('me:gift') : t('me:donation')
+        } . ${record.details.project}`;
```

Comment:
Should this be a dot? or a bullet point?
I think bullet point will look much better.

Reply:
Done in fd94e4c